File: Basics_with_JavaScript/78 Conditional Statements Advanced/More Exersises/05. Vacation.js
function vladi(input){

    let budged = Number(input[0]);
    let season = input[1];
    let location = "";
    let typeHouse = "";
    let price = 0;
    let pricePercentFromBudged = 0

    if (budged <= 1000) {
        typeHouse = "Camp";
        switch (season) {
            case "Summer":
                location = "Alaska";
                pricePercentFromBudged = 65;
                break;
            case "Winter":
                location = "Morocco";
                pricePercentFromBudged = 45
                break;
            default:
                break;
        }
    } else if (budged > 100 && budged <= 3000) {
        typeHouse = "Hut";
        switch (season) {
            case "Summer":
                location = "Alaska";
                pricePercentFromBudged = 80    
                break;
            case "Winter":
                location = "Morocco";
                pricePercentFromBudged = 60;
                break;
            default:
                break;
        }
    } else if (budged > 3000) {
        typeHouse = "Hotel";
        pricePercentFromBudged = 90
        switch (season) {
            case "Summer":
                location = "Alaska";
                break;
            case "Winter":
                location = "Morocco";
                break;
            default:
                break;
        }
    }

    
    price = (budged * (pricePercentFromBudged / 100)).toFixed(2)
    console.log(`${location} - ${typeHouse} - ${price}`);
    }
   
    vladi(["800", "Summer"]);
    vladi(["799.50", "Winter"]);
    vladi(["3460", "Summer"]);
    vladi(["1100", "Summer"]);
    vladi(["5000", "Winter"]);
    vladi(["2543.99", "Winter"]);

File: Basics_with_Python/PythonBasicCourse/3_Conditional_Statements_Advanced/8_Conditional_Statements_Advanced_Exercise_Lab/more_exercises/05._Vacation.py
budged = float(input())
season = input()
price = 0
place = 0
location =0

if budged <= 1000:
    place = "Camp"
    if season == "Summer":
        price = budged * 0.65
        location = "Alaska"
    elif season == "Winter":
        location = "Morocco"
        price = budged * 0.45
elif 1000 < budged <=3000:
    place = "Hut"
    if season == "Summer":
        location = "Alaska"
        price = budged * 0.80
    elif season == "Winter":
        location = "Morocco"
        price = budged * 0.60

elif budged > 3000:
    place = "Hotel"
    price = budged * 0.9
    if season == "Summer":
        location = "Alaska"
    elif season == "Winter":
        location = "Morocco"
print(f"{location} - {place} - {price:.2f}")
